test/expected/bgw_launcher.out

```diff
@@ -658,7 +658,7 @@ SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE backend_type = 'Tim
 -- make sure nobody is using it
 REVOKE CONNECT ON DATABASE :TEST_DBNAME FROM public;
 CALL kill_database_backends(:'TEST_DBNAME');
-SELECT * FROM pg_stat_activity WHERE datname = :'TEST_DBNAME';
+SELECT * FROM pg_stat_activity WHERE datname = :'TEST_DBNAME' AND backend_type != 'TimescaleDB Background Worker Scheduler';
```

Comment:
The point of this query was to debug the subsequent failing DROP, because DROP fails with a message that is not very helpful. I see that here we're doing ALTER TABLESPACE instead, but isn't the code that prevents new connections the same? It's also REVOKE CONNECT + kill all backends. If this fails to stop the scheduler, we're going to have problems with DROP again, right?

Reply:
Make sense. Actually this debug is unecessary since we do it a bit later in the test. Reworked it a bit and introduced `timescaledb_pre_restore()` insted of `stop_background_workers()`.